### PR TITLE
test file from Nina added

### PR DESCRIPTION
**Pull request by jolivuk**
Commit from Nina

### CODE REVIEW

**Comment by maksym-glushenkov** [comment]
good

**Review by marinaniva — APPROVED** [review]
+

**Comment by jolivuk** [comment]
Thank you

File: HackerRankIssues/Nina_Directory/test [hunk 1]
```diff
@@ -0,0 +1 @@
+My test file
```
Comment (maksym-glushenkov) [inline HackerRankIssues/Nina_Directory/test:1]: Ok passt

**Review by maksym-glushenkov — APPROVED** [review]

**Pull request merged**